test/game_stats_test.rb

```diff
@@ -28,15 +28,48 @@ def test_attributes_for_instance_of_game_withing_game_stats
     assert_equal 3, @game.home_goals
   end
 
+  def test_it_can_calculate_percentage_ties
+    assert_equal 14.29, @games_stats.percentage_ties
+  end
+
+  def test_it_can_calculate_percentage_home_wins
+    assert_equal 57.14, @games_stats.percentage_home_wins
+  end
+
+  def test_it_can_count_games_by_season
+    games_by_season = {
+      "20152016" => 3,
+      "20132014" => 2,
+      "20142015" => 1,
+      "20162017" => 1
+    }
```

Comment:
We can refactor this later to match the hash in test_it_can_calculate_average_goals_per_season.